src/Data/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2008, 2009, Nicolas Pouillard
```

Comment:
I think it would be better to put this license text in a comment at the top of `BoolExpr.hs`.  That way it's clearer what the license applies to, and there won't be any confusion if we later add more `Data.XXX` modules (unlikely, but possible).

Reply:
done